src/System.Net.Http/tests/FunctionalTests/PlatformHandlerTest.cs

```diff
@@ -55,6 +58,24 @@ public async Task GetAsync_TrailingHeaders_Ignored(bool includeTrailerHeader)
                 }
             });
         }
+
+
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Specifying Version(2,0) throws exception on netfx")]
+        [Fact]
+        public async Task SendAsync_RequestVersion20_HttpNotHttps_NoUpgradeRequest()
+        {
+            await LoopbackServer.CreateClientAndServerAsync(async uri =>
+            {
+                using (HttpClient client = CreateHttpClient())
+                {
+                    (await client.SendAsync(new HttpRequestMessage(HttpMethod.Get, uri) { Version = new Version(2, 0) })).Dispose();
+                }
+            }, async server =>
+            {
+                List<string> headers = await server.AcceptConnectionSendResponseAndCloseAsync();
+                Assert.All(headers, header => Assert.DoesNotContain("Upgrade", header, StringComparison.OrdinalIgnoreCase));
+            });
+        }
```

Comment:
Why did this move here?  Don't we still want to test this for SocketsHttpHandler in the default mode where the switch/envvar aren't set?

Reply:
I moved it in first iteration and forget to move it back. fixed now.